src/AppInstallerCommonCore/Public/winget/ExperimentalFeature.h

```diff
@@ -48,6 +49,9 @@ namespace AppInstaller::Settings
         ExperimentalFeature& operator=(ExperimentalFeature&&) = default;
 
         static bool IsEnabled(Feature feature);
+        // For unit testing
```

Comment:
>// For unit testing [](start = 8, length = 19)

We have a macro that we wrap around these, if for no other reason than to track them.

```
#ifndef AICLI_DISABLE_TEST_HOOKS
```